eras/babbage/impl/cardano-ledger-babbage.cabal

```diff
@@ -60,7 +60,7 @@ library
         cardano-crypto-class,
         cardano-data >=1.0,
         cardano-ledger-allegra >=1.1,
-        cardano-ledger-alonzo ^>=1.4,
+        cardano-ledger-alonzo ^>=1.4.1,
         cardano-ledger-binary >=1.0,
         cardano-ledger-core >=1.5 && <1.7,
```

Comment:
Same question as for alonzo cabal file, shouldn't this depend on cardano-ledger-core >= 1.6 ?

Reply:
:100: 

Fixed